OsmoSystems/osmo-jupyter
# Blackify osmo-jupyter

[asana ticket](https://app.asana.com/0/819671808102776/1127643074336798/f)

I did the following:

* checkout latest master
* create f/black-formatter branch
* run "black ."
* Review diff, use commented fmt: blocks and roll back anything egregious (search for `fmt` for how this looks. unfortunately I couldn't figure out a way to turn formatting off and comment it on one line)
* edit test_requirements.txt to add black, pytest-black
* edit setup.cfg to add flake8 ignores and --black pytest flag,
* edit README.md to add "code style: black" flag
* check that `pytest` is happy and that if I mess up formatting, `pytest` now displays the diff that `black` desires after all the test output
* commit
* open PR

## TODO:

same thing for cosmobot-process-experiment and cosmobot-run-experiment if this passes review
update code style manifesto if this passes review
